client/src/components/Modal.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
```

Comment:
this might be removed when nate do material on this since material has a dialog component already. Just letting you know